remo/featuredrep/admin.py

```diff
@@ -25,5 +25,13 @@ class FeaturedRepAdmin(ExportMixin, admin.ModelAdmin):
     def featured_rep_users(self, obj):
         return ', '.join([user.get_full_name() for user in obj.users.all()])
 
+    def save_model(self, request, obj, form, change):
+        super(FeaturedRepAdmin, self).save_model(request, obj, form, change)
+        featured = FeaturedRep.objects.get(pk=obj.pk)
+
+        if (form.is_valid() and 'updated_on' in form.changed_data and
```

Comment:
This method is only triggered through the admin interface. The model instance `obj` is given to this method. If the admin form is valid and the form has changed data in it, then the save method from  the model will be called with extra kwargs.

The line 34 in models will check if there are any extra kwargs. If not, it will update the `updated_on` field with the `now()` date. If there are - passed only through the admin - then it will use the custom date. At least that's what I was trying to achieve. Please keep asking - we will have fewer mistakes this way :)